lib/src/rdb/managers.rs

```diff
@@ -8,7 +8,7 @@ use crate::errors::Result;
 use crate::models;
 use crate::util;
 
-use rocksdb::{ColumnFamily, DBIterator, Direction, IteratorMode, WriteBatch, DB};
+use rocksdb::{ColumnFamilyRef, DBIterator, Direction, IteratorMode, WriteBatch, DB};
```

Comment:
The changes in this file aren't necessary any longer, is there merit to keeping them?

Reply:
It's a utility trait accept both supported references to `&ColumnFamily` and `BoundColumnFamily`. Maybe one day or one scenario MultiThreaded DB runs faster than SingleThread, these changes will make easy switching between two, or co-existing. And I believe that keeping them wont slow down the program.

bench:
indradb-latest:
```
test memory::bench_bulk_insert    ... bench:   5,030,010 ns/iter (+/- 14,085)
test memory::bench_create_edge    ... bench:          30 ns/iter (+/- 0)
test memory::bench_create_vertex  ... bench:         160 ns/iter (+/- 8)
test memory::bench_get_edge_count ... bench:          48 ns/iter (+/- 0)
test memory::bench_get_edges      ... bench:         108 ns/iter (+/- 1)
test memory::bench_get_vertices   ... bench:          96 ns/iter (+/- 3)
test rdb::bench_bulk_insert       ... bench:  57,581,086 ns/iter (+/- 16,077,371)
test rdb::bench_create_edge       ... bench:       4,457 ns/iter (+/- 91)
test rdb::bench_create_vertex     ... bench:       2,909 ns/iter (+/- 203)
test rdb::bench_get_edge_count    ... bench:       1,093 ns/iter (+/- 11)
test rdb::bench_get_edges         ... bench:         545 ns/iter (+/- 11)
test rdb::bench_get_vertices      ... bench:         678 ns/iter (+/- 18)
```

with changes:
```
test memory::bench_bulk_insert    ... bench:   5,021,918 ns/iter (+/- 13,279)
test memory::bench_create_edge    ... bench:          30 ns/iter (+/- 8)
test memory::bench_create_vertex  ... bench:         163 ns/iter (+/- 8)
test memory::bench_get_edge_count ... bench:          48 ns/iter (+/- 0)
test memory::bench_get_edges      ... bench:         112 ns/iter (+/- 1)
test memory::bench_get_vertices   ... bench:          98 ns/iter (+/- 0)
test rdb::bench_bulk_insert       ... bench:  56,649,500 ns/iter (+/- 16,651,508)
test rdb::bench_create_edge       ... bench:       4,204 ns/iter (+/- 86)
test rdb::bench_create_vertex     ... bench:       2,864 ns/iter (+/- 172)
test rdb::bench_get_edge_count    ... bench:       1,073 ns/iter (+/- 14)
test rdb::bench_get_edges         ... bench:         546 ns/iter (+/- 24)
test rdb::bench_get_vertices      ... bench:         671 ns/iter (+/- 7)
```

Reply:
Yeah it's alias for the same thing. Fair enough, we can keep this.